includes/wcexd-products-download.php

```diff
@@ -289,10 +289,14 @@ function wcexd_products_download() {
 				}
 
 				$tax_rate = 1 === intval( $wcexd_products_tax_name ) ? WCtoDanea::get_tax_rate( $product->get_id(), 'name' ) : WCtoDanea::get_tax_rate( $product->get_id() );
+				
+				$variation = new WC_Product_Variation($product_id);
```

Comment:
Occorre verificare che il prodotto in questione sia in effetti una variazione o il risultato sarà un _Fatal error_.

Reply:
Non ho aggiunto controlli sul tipo di prodotto perché nel test misto non ha generato errori, così ho dato per scontato che la variabile non venisse popolata, ma molto più safe con il codice che hai aggiunto, inoltre ha un aspetto più gradevole 🖖🏻